aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -166,11 +166,17 @@ def save(
                 channel_colors=channel_colors,
                 dimension_order=dimension_order,
             )
+            xml = self.omeMetadata.to_xml().encode()
```

Comment:
previously, that line was being done once regardless of the if/else cases, at the bottom.  now it has to be different for one of the cases because I don't want to let the string be passed through the OMEXML class